experimenter/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.tsx

```diff
@@ -19,6 +19,7 @@ import {
   TOOLTIP_DURATION,
 } from "src/lib/constants";
 import { getStatus } from "src/lib/experiment";
+import "src/styles/index.scss";
```

Comment:
you will also need to remove this import

Reply:
Mam done with removing the imports